app/controllers/authentication/user_authenticator.rb

```diff
@@ -52,6 +52,10 @@ def user_logged_in?
     raise NotImplementedError, 'implement in subclass'
   end
 
+  def logged_out_path
```

Comment:
:+1:

Reply:
make session_new_path default and override for keyloak ?

Reply:
:+1: